tmt/schemas/prepare/errata.yaml

```diff
@@ -0,0 +1,46 @@
+---
+
+#
+# JSON Schema definition for `errata` prepare plugin
+#
+# Private plugin usable only within RH
+#
+
+$id: /schemas/prepare/errata
+$schema: https://json-schema.org/draft-07/schema
+
+type: object
+additionalProperties: false
+
+properties:
+
+  how:
+    type: string
+    enum:
+      - errata
+
+  errata:
+    oneOf:
+      - anyOf:
+          - type: string
+            pattern: "^[0-9]+$"
```

Comment:
Is leading zero really acceptable?

Reply:
I see no reason why not, leading '0' should not make it problematic wrt int base 10 conversion any more. Easier to copy&paste from logs where one might find id filled with leading zeros.

Reply:
Ok, makes sense, and it's true `int()` won't stumble.